_includes/episode_navbar.html

```diff
@@ -12,8 +12,27 @@
 <div class="row">
   <div class="col-xs-1">
     <h3 class="text-left">
-      {% if page.previous.url %}
-      <a href="{{ relative_root_path }}{{ page.previous.url }}"><span class="glyphicon glyphicon-menu-left" aria-hidden="true"></span><span class="sr-only">previous episode</span></a>
+      {% assign prev_epi = "NONE" %}
+      {% assign prev_url = false %}
+      {% assign prev = false %}
+      {% for epi in site.contents.episodes %}
+        {% if epi == page.slug %}
+          {% if prev_epi %}
+            {% assign prev_epi = prev %}
+          {% endif %}
+          {% break %}
+        {% else %}
+          {% assign prev = epi %}
+        {% endif %}
+      {% endfor %}
+      {% for episode in site.episodes %}
+        {% if prev_epi == episode.slug %}
+          {% assign prev_url = episode.url %}
+          {% break %}
+        {% endif %}
+      {% endfor %}
+      {% if prev_url %}
```

Comment:
I think it would be better to move this to a separate file and then be included in this file... Just like `base_path.html` above.
@fmichonneau, please 👍  if you agree.